### PR TITLE
work with form

### PR DESCRIPTION
**Pull request by RuslanKhoroshylov**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,121 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <label class="field">
+            Surname:
+          <input type="text" autocomplete="off">
```
Comment (vlad-demchuk) [inline src/index.html]:
```suggestion
        <label class="field">
          Surname:
          <input type="text" autocomplete="off">
```

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,121 @@
...
+        </label>
+        <label class="field">
+            Name:
+          <input type="text" autocomplete="off">
```
Comment (vlad-demchuk) [inline src/index.html]:
```suggestion
          Name:
          <input type="text" autocomplete="off">
```
no need to add these indents, fix all cases

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,121 @@
...
+        <label class="field">
+            How old are You?
+          <input 
+          type="number" 
+          name="how old are you" 
+          placeholder="Your Age" 
+          value="12" 
+          min="1" 
+          max="100"
+          >
```
Comment (vlad-demchuk) [inline src/index.html]: check your code style
image

Comment (vlad-demchuk) [reply src/index.html]: fix all such cases

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
add links to PR description

File: src/index.html (hunk 3 above)
Comment (RuslanKhoroshylov) [reply src/index.html]: all changes complete

**Review by etojeDenys — APPROVED** [review]
looks good, but add DEMO, Report link

**Review by etojeDenys — CHANGES_REQUESTED** [review]
:)

**Comment by RuslanKhoroshylov** [comment]
не пойму что за DEMO и где ее взять

**Pull request closed without merging**